lib/matplotlib/collections.py

```diff
@@ -1827,12 +1828,20 @@ def determine_facecolor(patch):
                 if patch.get_fill():
                     return patch.get_facecolor()
                 return [0, 0, 0, 0]
-
+            def is_default(c):
```

Comment:
This in only the default default (that is, there is a set of  rcparams which will cause this to be miss.  Looking at https://github.com/matplotlib/matplotlib/blob/c19cd343742f9f3a97e453b3072e4a5794491794/lib/matplotlib/patches.py#L315-L339 we have some private state (which we are allowed to use inside of Matplotlib) to check this.  In this case, I think it is better to check `p._original_edgecolor` and if `None` either re-implement the default logic or pass in `None` here (if we can) and let the next layer down deal with resolving the default behavior.